### DISCUSSION TITLE
Adding Fleet node

### DISCUSSION THREAD

**Discussion by Sahale** [Security-Onion-Solutions/securityonion#5431] [Unsupported Versions] [answered]
Hello.
SO 2.3.70, installed on clean CentOS.
I've added Fleet node to my working deployment. I can see node in Grafana, etc, but I can't access Fleet via web-interface: there is no link from manager node default web page, and if I try to connect to fleet node directly, it redirects me to manager node.I didn't install Fleet in manager node previously.
I've done something wrong? How to access Fleet?
Thanks!

**Comment by defensivedepth** [comment, marked as answer]
This should not be happening, will check into what is going on and let you know.

**Reply by defensivedepth** [reply]
I have merged in a fix for this bug - You can see the changes here: https://github.com/Security-Onion-Solutions/securityonion/pull/5508/files

**Reply by Sahale** [reply]
Super, thanks a lot! Should I just reinstall fleet node?

**Reply by defensivedepth** [reply]
You could actually just make the change in the following file (on the Manager):

`/opt/so/saltstack/default/salt/soc/files/soc/soc.json

And then restart SOC: `sudo so-soc-restart`

And that should fix it, you should see the Link in SOC.
